.github/codecov.yml

```diff
@@ -33,22 +33,22 @@ ignore:
   - "test"
 
 flags:
-  unittests-Solution-Stable:
+  unittests-Solution:
     carryforward: true
     paths:
       - src
 
-  unittests-Solution-Experimental:
+  unittests-Project-Stable:
     carryforward: true
     paths:
       - src
 
-  unittests-Instrumentation-Stable:
+  unittests-Project-Experimental:
     carryforward: true
     paths:
       - src
 
-  unittests-Instrumentation-Experimental:
+  unittests-UnstableCoreLibraries-Experimental:
```

Comment:
Is there still a concept called "core libraries" after instrumentation libs moved away?

Reply:
There's two tags:

`<MinVerTagPrefix>core-</MinVerTagPrefix>`
`<MinVerTagPrefix>coreunstable-</MinVerTagPrefix>`

But I get your point, we could probably just use `stable-` and `unstable-` since everything is now "core" 🤣 Follow-up!